grpc/Makefile

```diff
@@ -1,11 +1,11 @@
 # Copyright (2019) Cobalt Speech and Language Inc.
 
-.PHONY: all go py cs swift cs_push android go-test py-test docs
+.PHONY: all go py cs swift cs_push android java-lite java go-test py-test docs
 
 all: docs go py swift
 
 ifeq ($(PROTOINC),)
- $(error "env variable PROTOINC must be defined to the location of googleapis proto files")
+# $(error "env variable PROTOINC must be defined to the location of googleapis proto files")
```

Comment:
Do you mean to leave this commented out?

Reply:
No sorry.